src/main/java/org/opensearchmetrics/lambda/GithubEventsLambda.java

```diff
@@ -0,0 +1,89 @@
+            "issues.assigned",
+            "issue_comment.created",
+            "pull_request.closed",
+            "pull_request.opened",
+            "pull_request.labeled",
+            "pull_request.unlabeled",
+            "pull_request.assigned",
+            "pull_request_review.submitted",
+            "pull_request_review_comment.created",
+            "gollum"};
+    private static final ServiceComponent COMPONENT = DaggerServiceComponent.create();
+    private final OpenSearchUtil openSearchUtil;
+    private final S3Util s3Util;
+    private final ObjectMapper mapper;
+
+    public GithubEventsLambda() {
+        this(COMPONENT.getOpenSearchUtil(), COMPONENT.getS3Util(), COMPONENT.getObjectMapper());
+    }
+
+    @VisibleForTesting
+    GithubEventsLambda(@NonNull OpenSearchUtil openSearchUtil, @NonNull S3Util s3Util, @NonNull ObjectMapper mapper) {
+        this.openSearchUtil = openSearchUtil;
+        this.s3Util = s3Util;
+        this.mapper = mapper;
+    }
+
+    @Override
+    public Void handleRequest(Void input, Context context) {
+        final String yesterday = LocalDate.now(ZoneOffset.UTC).minus(1, ChronoUnit.DAYS).toString(); // yyyy-MM-dd
+        Map<String, String> finalEventData = new HashMap<>();
+        for (String eventToIndex : eventsToIndex) {
+            String prefix = eventToIndex + "/" + yesterday + "/";
+            List<String> objectKeys = s3Util.listObjectsKeys(prefix);
+            for (String objectKey : objectKeys) {
+                try (ResponseInputStream<GetObjectResponse> eventInputStream = s3Util.getObjectInputStream(objectKey)) {
+                    JsonNode eventNode = mapper.readTree(eventInputStream);
+                    EventData event = new EventData();
+                    event.setId(eventNode.path("id").textValue());
+                    event.setType(eventNode.path("name").textValue());
+                    event.setRepository(eventNode.path("payload").path("repository").path("name").textValue());
+                    event.setOrganization(eventNode.path("payload").path("organization").path("login").textValue());
+                    if (event.getOrganization() == null) {
+                        event.setOrganization(eventNode.path("payload").path("repository").path("owner").path("login").textValue());
+                    }
+                    event.setAction(eventNode.path("payload").path("action").textValue());
+                    event.setSender(eventNode.path("payload").path("sender").path("login").textValue());
+                    event.setCreatedAt(eventNode.path("uploaded_at").textValue());
+
+                    finalEventData.put(event.getId(), event.getJson(event, mapper));
+                } catch (Exception e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        }
+        openSearchUtil.createIndexIfNotExists("github-events");
+        openSearchUtil.bulkIndex("github-events", finalEventData);
```

Comment:
Please add as `github-events-MM-YYY"`, this way we can clean up older data easily if required.